src/domain/eventSearch/constants.ts

```diff
@@ -100,18 +122,33 @@ export const SPORT_KEYWORDS = [
   'yso:p965', // Urheilu
 ];
 
-export const MAPPED_CATEGORIES: Record<string, string> = {
-  [CATEGORIES.CULTURE]: CULTURE_KEYWORDS.join(','),
-  [CATEGORIES.DANCE]: 'yso:p1278',
-  [CATEGORIES.FOOD]: 'yso:p3670',
-  [CATEGORIES.INFLUENCE]: INFLUENCE_KEYWORDS.join(','),
-  [CATEGORIES.MISC]: 'yso:p2108',
-  [CATEGORIES.MOVIE]: 'yso:p1235',
-  [CATEGORIES.MUSEUM]: MUSEUM_KEYWORDS.join(','),
-  [CATEGORIES.MUSIC]: 'yso:p1808',
-  [CATEGORIES.NATURE]: 'yso:p2771',
-  [CATEGORIES.SPORT]: SPORT_KEYWORDS.join(','),
-  [CATEGORIES.THEATRE]: 'yso:p2625',
+export const MAPPED_EVENT_CATEGORIES: Record<string, string> = {
+  [EVENT_CATEGORIES.CULTURE]: CULTURE_KEYWORDS.join(','),
+  [EVENT_CATEGORIES.DANCE]: 'yso:p1278',
+  [EVENT_CATEGORIES.FOOD]: 'yso:p3670',
+  [EVENT_CATEGORIES.INFLUENCE]: INFLUENCE_KEYWORDS.join(','),
+  [EVENT_CATEGORIES.MISC]: 'yso:p2108',
+  [EVENT_CATEGORIES.MOVIE]: 'yso:p1235',
+  [EVENT_CATEGORIES.MUSEUM]: MUSEUM_KEYWORDS.join(','),
+  [EVENT_CATEGORIES.MUSIC]: 'yso:p1808',
+  [EVENT_CATEGORIES.NATURE]: 'yso:p2771',
+  [EVENT_CATEGORIES.SPORT]: SPORT_KEYWORDS.join(','),
+  [EVENT_CATEGORIES.THEATRE]: 'yso:p2625',
+};
+
+//todo: replace with valid keyword ids
+export const MAPPED_COURSE_CATEGORIES: Record<string, string> = {
+  [EVENT_CATEGORIES.MOVIE]: 'yso:p1235',
+};
+
+export const MAPPED_CATEGORIES: Record<string, Record<string, string>> = {
+  [String(EVENT_SEARCH_SOURCES.COURSES)]: MAPPED_COURSE_CATEGORIES,
+  [String(EVENT_SEARCH_SOURCES.EVENTS)]: MAPPED_EVENT_CATEGORIES,
+};
+
+export const MAPPED_CATEGORY_TERMS: Record<string, string> = {
+  [String(EVENT_SEARCH_SOURCES.COURSES)]: 'keyword_set_AND',
+  [String(EVENT_SEARCH_SOURCES.EVENTS)]: 'keyword',
```

Comment:
is String() function redundant here? Nothing seems to break when I remove it

Reply:
actually without it typescript gives error

Reply:
hmm has it something to do with typescript version? for me there we no errors

Reply:
hmm, you are right. now its not giving error, maybe during development i had mistake somewhere else